juriscraper/opinions/united_states/state/ill.py

```diff
@@ -42,20 +47,22 @@ def _process_html(self) -> None:
         """
         for row in self.html.xpath("//table[@id='ctl04_gvDecisions']/tr"):
             cells = row.xpath(".//td")
-
             # Don't parse pagination rows or headers or footers
             if len(cells) != 7 or row.xpath(".//table"):
                 continue
-
-            name = get_row_column_text(row, 1)
-            date = get_row_column_text(row, 3)
-            citation = get_row_column_text(row, 2)
             try:
                 url = get_row_column_links(row, 1)
             except IndexError:
                 # If the opinion file's information is missing (as with
                 # links to withdrawn opinions), skip record
                 continue
```

Comment:
Should we add a logger warning here too? or do we not because we assume we skip the link because its been removed.

Reply:
Added informative log.
Doesn't hurt to have additional information about those exceptions.